.github/workflows/check_scala3_nightly.yml

```diff
@@ -1,22 +0,0 @@
-name: "Check Scala nightly release"
-on:
-  workflow_dispatch:
-  schedule:
-    - cron: 0 5 * * *
-jobs:
-  test_and_release:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-java@v3
-        with:
-          distribution: "temurin"
-          java-version: "17"
-      - name: "Query maven-central"
-        run: |
-          sbt 'save-non-published-nightlies versions'
```

Comment:
Can we also just remove the `save-non-published-nightlies` command in the build then as well?

Reply:
Yep, done!